commit db11f62fcbc7bb4333c026853928d8a1744f2799
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2019-03-30T01:45:33+00:00 (6 months ago)
    
    OS-7327 Want connstat(1M) command to display per-connection TCP statistics (fix timestamp fallthrough)

diff --git a/usr/src/cmd/connstat/connstat_main.c b/usr/src/cmd/connstat/connstat_main.c
index 8fa8dc4fb7..dd58e2ac2b 100644
--- a/usr/src/cmd/connstat/connstat_main.c
+++ b/usr/src/cmd/connstat/connstat_main.c
@@ -461,6 +461,7 @@ print_timestamp(timestamp_fmt_t timestamp_fmt, boolean_t parsable)
 		if (len > 0) {
 			(void) printf("%s%s\n", pfx, dstr);
 		}
+		break;
 	}
 	default:
 		abort();
